exercises/bank-account/Hints.md

```diff
@@ -0,0 +1,4 @@
+## Hints
+This exercise requires you to handling data related to currency and money. A normal approuch is to use the [Decimal] (https://msdn.microsoft.com/en-US/library/system.decimal.aspx) struct to store currency values. But this only stores the numeric of a currency. 
```

Comment:
"But this only stores the numeric of a currency." should be "Note though that you then only store the numeric value of a currency."